dashboard/src/t5gweb/api.py

```diff
@@ -67,6 +67,9 @@ def refresh(data_type):
     elif data_type == "issues":
         get_issue_details(cfg)
         return jsonify({"caching issues": "ok"})
+    elif data_type == "newcases":
```

Comment:
Suggest renaming this to something that makes it clear that Jira cards will be created (maybe `create_jira_cards` or `sync_portal_to_jira`. Most of the other endpoints are read-only, so I think it's worth making a distinction